packages/wunderctl/src/binarypath.ts

```
@@ -52,5 +52,8 @@ export const wunderctlName = (): string => {
 export const binaryDir = path.join(__dirname, '..', 'download');
 
 export const wunderctlBinaryPath = (): string => {
+	if (process.env.WUNDERCTL_BINARY_PATH) {
+		return process.env.WUNDERCTL_BINARY_PATH;
```

Should we validate for a valid path? I could imagine the error message is not very helpful when the path does not point to a binary.

---

At least for now, we're the only ones setting that variable. If someone is setting it manually before invocation I think letting it blow up makes it more evident than swallowing the error. There's also already a check in `wunderCtlFile()` which throws if the binary doesn't exist, so I'd say there's already enough validation.

---

ok